src/renderer/coremods/badges/badge.tsx

```diff
@@ -71,108 +64,137 @@ export const getBadgeComponent = async (): Promise<(args: BadgeProps) => React.R
   }
   const { profileBadge22 } = badgeClassMod;
 
-  return ({ color, tooltip, tooltipPosition, className, children, gap, onClick }: BadgeProps) => {
-    return (
-      <Clickable className={clickableClass.clickable} onClick={onClick || (() => undefined)}>
+  const child = (
+    <div
+      className={`${profileBadge22} replugged-badge ${className || ""}`}
+      style={{ color: `#${color || "7289da"}` }}>
+      {children}
+    </div>
+  );
+  return (
+    <Clickable className={clickableClass.clickable} onClick={onClick || (() => undefined)}>
```

Comment:
If onClick isn't defined, then the badge shouldn't be clickable

Reply:
The clickable class gives it the cursor, which I think it should have due to the tooltip.

Reply:
I meant that if onclick isn't defined, then the cursor shouldn't be the clickable type of cursor lol. If it doesn't do anything, there shouldn't be any need to have the clickable cursor.

Reply:
All of the Discord badges show a pointer even if they don't do anything onclick.